.idea/runConfigurations.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RunConfigurationProducerService">
+    <option name="ignoredProducers">
+      <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
```

Comment:
Why are we using something from android?

Reply:
I'm not 100% sure. IntelliJ automatically created these files when I created the run configurations through the GUI. I tried to look up this value, but I couldn't find anything. I'll try removing this file and see if the run configurations are still functional.

Reply:
I tried removing this file from my project and IntelliJ just regenerated it. I tried removing sections of the xml to see if I could remove the `com.android.tools` line, but either the file was regenerated, or IntelliJ's Android Plugin threw an error. The Android Plugin is bundled with IntelliJ. My best guess as to what this line indicates is that the Android Plugin's Run Configuration Producers should ignore this project, since this is not an Android project.

See this excerpt from the JetBrains MPS (Meta-Programming System) documentation for more information:

> Producers for a run configuration describe how to create this configuration for various nodes or groups of nodes, models, modules or a project. This makes run configurations easily discoverable for users since for each producer they will see an action in the context menu suggesting to run the selected item. Also this simplifies configuring because it gives a default way to execute something without seeing the editing dialog first.

Source: [JetBrains MPS - Producers](https://www.jetbrains.com/help/mps/run-configurations.html#producers)
See Also: [JetBrains MPS - Sharing Run Configurations](https://www.jetbrains.com/help/mps/run-debug-configuration.html#share-configurations)

I can remove this file from the PR at your request, but I do not think it will matter if it is included or not.

Reply:
@JeremyEastham please also modify the .gitignore file to remove this xml file from git

Reply:
It has been added.